07 Form/src/components/member/memberPage.tsx

```diff
@@ -17,7 +16,7 @@ interface State {
 }
 
 // Nice tsx guide: https://github.com/Microsoft/TypeScript/wiki/JSX
-export default class memberPage extends React.Component<Props, State> {
+export default class MemberPage extends React.Component<Props, State> {
 
   constructor(props : Props){
         super(props);
```

Comment:
Remove comments in line [34](https://github.com/Lemoncode/react-typescript-samples/pull/36/files#diff-34f4a408b20fac786fd9719184ac0ac3R34) and [35](https://github.com/Lemoncode/react-typescript-samples/pull/36/files#diff-34f4a408b20fac786fd9719184ac0ac3R35)